R/transform.R

```diff
@@ -4,11 +4,17 @@
 #'   the result.
 #' @param files A character vector with paths to the file that should be
 #'   transformed.
+#' @param assert_code Boolean to indicate whether checking for parsing errors
+#'   should be done prior to styling.
 #' @inheritParams make_transformer
 #' @return A logical value that indicates whether or not any file was changed is
 #'   returned invisibly. If files were changed, the user is informed to
 #'   carefully inspect the changes via a message sent to the console.
-transform_files <- function(files, transformers) {
+transform_files <- function(files, transformers, assert_code = TRUE) {
```

Comment:
In my eyes, there is one more tweak necessary in the `utf8` package. Unfortunately, it is not visible in the reprex above because `reprex` strangely does not show all warnings I get in the console. I opened [#15](https://github.com/krlmlr/utf8/issues/15) and submitted a [PR](https://github.com/krlmlr/utf8/pull/16).